lib/DDG/Spice/BrainyQuote.pm

```diff
@@ -17,7 +17,7 @@ handle remainder => sub {
     }
     
     # Avoid triggering on 'stock' quotes; these are handled by Stocks IA
-    if ($req->query_lc =~ m/stock quote/) {
+    if ($req->query_lc =~ m/stock quote/ || $req->query_lc =~ m/quote of the day/ || $req->query_lc =~ m/quote for the day/ ) {
```

Comment:
I think this should be moved out seeing as this section deals with 'stock' quotes.

So have a separate comment explaining this stops it conflicting with QuoteOfTheDay and have a separate condition.